src/test/components/TrackProcessCPU.test.js

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import type { IndexIntoSamplesTable, CssPixels } from 'firefox-profiler/types';
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { fireEvent } from '@testing-library/react';
+
+import { render } from 'firefox-profiler/test/fixtures/testing-library';
+import { TrackProcessCPU } from '../../components/timeline/TrackProcessCPU';
+import { ensureExists } from '../../utils/flow';
+
+import {
+  autoMockCanvasContext,
+  flushDrawLog,
+} from '../fixtures/mocks/canvas-context';
+import { mockRaf } from '../fixtures/mocks/request-animation-frame';
+import { storeWithProfile } from '../fixtures/stores';
+import {
+  addRootOverlayElement,
+  removeRootOverlayElement,
+  getMouseEvent,
+} from '../fixtures/utils';
+import {
+  getProfileFromTextSamples,
+  getCounterForThreadWithSamples,
+} from '../fixtures/profiles/processed-profile';
+import { autoMockElementSize } from '../fixtures/mocks/element-size';
+import { autoMockIntersectionObserver } from '../fixtures/mocks/intersection-observer';
+
+// The following constants determine the size of the drawn graph.
+const SAMPLE_COUNT = 8;
+const PIXELS_PER_SAMPLE = 10;
+const GRAPH_WIDTH = PIXELS_PER_SAMPLE * SAMPLE_COUNT;
+const GRAPH_HEIGHT = 10;
+
+function getSamplesPixelPosition(
+  sampleIndex: IndexIntoSamplesTable,
+  samplePosition
+): CssPixels {
+  // Compute the pixel position of the center of a given sample.
+  return sampleIndex * PIXELS_PER_SAMPLE + PIXELS_PER_SAMPLE * samplePosition;
+}
+
+/**
+ * This test verifies that the process CPU track can draw a graph of the process CPU.
+ */
+describe('TrackProcessCPU', function () {
+  function setup() {
+    const { profile } = getProfileFromTextSamples(
+      Array(SAMPLE_COUNT).fill('A').join('  ')
+    );
+    const threadIndex = 0;
+    const thread = profile.threads[threadIndex];
+    profile.counters = [
+      getCounterForThreadWithSamples(thread, threadIndex, {
+        time: thread.samples.time.slice(),
+        // CPU usage numbers for the per-process CPU.
```

Comment:
Adding `name` and `category` would be probably better than this comment, and also more correct (as I know we have a few `if` conditions checking them)

Reply:
Yeah, we are mostly checking them while we are enabling them in the console. Added the name and category.